pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaManaged.hpp

```diff
@@ -0,0 +1,1303 @@
+             applyCRY(std::forward<decltype(wires)>(wires),
+                      std::forward<decltype(adjoint)>(adjoint),
+                      std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"CRZ",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applyCRZ(std::forward<decltype(wires)>(wires),
+                      std::forward<decltype(adjoint)>(adjoint),
+                      std::forward<decltype(params[0])>(params[0]));
+         }},
+        // LCOV_EXCL_STOP
+        {"SingleExcitation",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applySingleExcitation(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"SingleExcitationPlus",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applySingleExcitationPlus(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"SingleExcitationMinus",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applySingleExcitationMinus(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"DoubleExcitation",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applyDoubleExcitation(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"DoubleExcitationPlus",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applyDoubleExcitationPlus(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"DoubleExcitationMinus",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applyDoubleExcitationMinus(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"ControlledPhaseShift",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applyControlledPhaseShift(
+                 std::forward<decltype(wires)>(wires),
+                 std::forward<decltype(adjoint)>(adjoint),
+                 std::forward<decltype(params[0])>(params[0]));
+         }},
+        {"Rot",
+         [&](auto &&wires, auto &&adjoint, auto &&params) {
+             applyRot(std::forward<decltype(wires)>(wires),
```

Comment:
It's a little bit strange since it's tested [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/6bc75d71a20fbbc2146cf69821e1304bc505fef2/pennylane_lightning/core/src/simulators/lightning_gpu/gates/tests/Test_StateVectorCudaManaged_Param.cpp#L410) but not captured by codecov. I think we can mark it tested. How do you think?

Reply:
Quite strange --- I'd like to resolve coverage issues, but if it is getting tested perfectly, then I think we can assume this is fine.